about.html

```diff
@@ -10,64 +10,144 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
+<!-- Web page starts here -->
 
-<body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
 
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
+<body>
+  <div class="container-about-page">
+    <header>
+        <section class="headerTop">
+            <div class="lambdaLogo">
+                <img src="https://github.com/LambdaSchool/Sprint-Challenge--User-Interface/blob/master/img/lambda-black.png?raw=true">
+            </div>
+            
+            <nav class="navBar">
+                <a href="index.html">Home</a>
+                <a href="about.html">About</a>
+                <a href="#">Products</a>
+                <a href="blog.html">Blog</a>
+                <a href="#">Contact</a>
+            </nav>
+        </section>  
+        <section class="headerBottom">
+            <img src="https://github.com/LambdaSchool/Sprint-Challenge--User-Interface/blob/master/img/jumbo-about.png?raw=true"
+        </section>  
+    </header>
     
-    Collaboration
+    <section class="aboutTopContent">
+      <h1>
+        About
+      </h1>
+  
+      <p>
+        Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+      </p>
+      
+
+    </section>
+
+    <section class="aboutMidContent">
```

Comment:
Start with flexbox and then go to margin / padding (you can see here while it is centered it look to have more white space on the left then the right.